capture/writer-simple.c

```diff
@@ -499,7 +504,11 @@ LOCAL void writer_simple_check(MolochSession_t *session, void *UNUSED(uw1), void
     if (now.tv_sec - lastSave[session->thread].tv_sec < 10)
         return;
 
-    writer_simple_process_buf(session->thread, 0);
+    if (config.maxFileTimeM > 0 && now.tv_sec - fileAge[session->thread].tv_sec >= config.maxFileTimeM*60) {
```

Comment:
I think you want to move this check above the 10 second idle check, otherwise it will only do this check if there have been no saves for 10 seconds.  Remove the else and leave that , 0 write down here though.

Reply:
Right. In the use case I was thinking about it didn't make a difference - avoiding situation, that the file is stuck unfinalized for days if there is no more traffic on the interface; the file can get removed by moloch viewer in the meantime and then new traffic is dropped on the floor. 

You are right, that in general, it should be checked anyway, so the rotation time is actually considered.
Fixed!